src/main/java/main.java

```diff
@@ -1,9 +1,26 @@
-import entities.User;
+import controllers.createUserController;
+import presenters.startPageUI.startPageViewMain;
+import presenters.startPageUI.startPageViewWelcome;
+import presenters.startPageUI.startPageViewModel;
+
+import javax.swing.*;
+import java.awt.*;
 
 public class main {
     public static void main(String[] args) {
-        User shaffaan = new User("shaffan", new String[]{"banana"});
-        System.out.println(shaffaan.getUsername());
+        JFrame application = new JFrame("RecipEZ");
+        application.setSize(550,200 );
+        application.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+        CardLayout cardLayout = new CardLayout();
+        JPanel screens = new JPanel(cardLayout);
+        application.add(screens);
+
+        createUserController mockRegisterController = new createUserController();
 
+        startPageViewModel startViewModel = new startPageViewModel(mockRegisterController);
```

Comment:
One comment startPageViewModelInterface it should

Reply:
startPageViewModelInterface startViewModel = new startPageViewModel(mockRegisterController);